<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Student Form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="student_done">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="gender">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Course Code</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="course_code">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="year_level">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Year Level</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>290</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="id_number">
       <property name="minimumSize">
        <size>
         <width>221</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>221</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Full Name     </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="full_name">
       <property name="minimumSize">
        <size>
         <width>221</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>221</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
